src/test/java/io/cryostat/net/web/http/api/v2/RulesPostHandlerTest.java

```diff
@@ -218,6 +218,28 @@ void unsupportedFirstMimeShouldThrow() {
             MatcherAssert.assertThat(ex.getStatusCode(), Matchers.equalTo(415));
         }
 
+        @ParameterizedTest
+        @CsvSource(
+              value = {
+	        "multipart/form-data",
+	        "multipart/form-data; boundary=------somecharacters",
+	        "multipart/form-data; unkown characters",
+	        "text/plain; another-directive"
```

Comment:
The `text/plain` case will fail because this handler doesn't actually accept that content type. You could move it out to another test with different assertions, or check if there is already another test that covers the same or similar case.

I think more cases like `multipart/form-data; directive1; directive2`, `multipart/form-data;directive` are worth testing as well, and they should ideally pass the same as the rest of your cases here.